workers/unity/Assets/Playground/Scripts/MonoBehaviours/TriggerShotBehaviour.cs

```diff
@@ -0,0 +1,38 @@
+using Improbable;
+using Improbable.Gdk.GameObjectRepresentation;
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Playground
+{
+    [WorkerType(WorkerUtils.UnityClient, WorkerUtils.AndroidClient, WorkerUtils.iOSClient)]
```

Comment:
This is client side authoritative then?

Reply:
Not with that attitude you can't!

Reply:
lol

Reply:
Server side authority isn't listening for input on the server, its not sending "trusted" commands from the client.

Reply:
Server side movement doesn't mean listening for keyboard presses.